es-MX/mozorg/mission.ftl

```diff
@@ -2,24 +2,37 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
Failed TM matching: This whole page has already been translated for a long time. Why were these 100% retranslated for mission.ftl?